### DISCUSSION TITLE
Any links for community chat like slack matrix or IRC ?

### DISCUSSION THREAD

**Discussion by suvanbanerjee** [openclimatefix/community#15] [New to OCF]
Hey openclimatefix community, I am student and a sometimes social worker from india currently in my junior year, Is there a way to connect with the community, like a calendar for meetings or a group chat?

**Comment by peterdudfield** [comment]
Hi @suvanbanerjee thanks so much getting in contact. We run OCF office hours every few months, and that might be a good way to connect. We do have a Community Slack group, but potentially it will be get re organised in the new year. In the mean time, github discussions are where we are pointing people towards

**Reply by suvanbanerjee** [reply]
@peterdudfield any updates about community Slack group ?

**Comment by Sagarpillai** [comment]
Hi @peterdudfield Im a UG student from India, I'd like to contribute and join OCF community, was wondering if  the slack group has reorganised, if yes, can I get the link to join it. Excited to contribute!!

**Reply by AUdaltsova** [reply]
Hi @Sagarpillai, we do not currently run a community slack. These discussions are our main community space at the moment!
